llmcord/llm_utils.py

```
@@ -0,0 +1,71 @@
+                    if res["content"].startswith("__STOP_RESPONSE__"):
+                        logger.debug("Iteration {}: Abort sentinel detected in tool results", i)
+                        return res["content"]
+
+                params["messages"].extend(results)
+                continue
+
+            if hasattr(message, "content") and message.content is not None:
+                return str(message.content)
+            return ""
+
+    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)